.github/workflows/update-contributors.yaml

```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches:
       - 'main'
+  schedule:
+  # Schedule at 9 AM JST (0 AM UTC) every Friday
+  - cron: '0 0 * * 5'
```

Comment:
![nits-badge](https://img.shields.io/badge/review-nits-green.svg) 
Thanks for your work!
I think you should lower the indent of comment and cron. [FYI](https://docs.github.com/ja/actions/using-workflows/events-that-trigger-workflows#schedule)

Reply:
Thank you for pointing that out! I completely missed it.
I fixed [391cb19](https://github.com/yumemi-inc/flutter-yumemi-lints/pull/56/commits/391cb19a92bff58f8396bcde79ae1f5aadcd39b9)

Reply:
Thanks for the correction!